app/root.css

```diff
@@ -267,6 +271,46 @@ svg * {
   filter: drop-shadow(0 4px 8px rgba(0, 0, 0, 0.1));
 }
 
+.menu {
+  border: 1px solid var(--borderColor);
+  border-radius: 5px;
+  overflow: hidden; /* needed to be able to round inner corner */
+  width: 11em;
+  position: absolute;
+  top: 9em;
+  right: -6em;
+  z-index: 10;
+}
+.fun-logo + .menu {
+  top: 14em;
+}
+
+.menu-link {
+  display: inline-block;
+  width: 100%;
+  padding: 10px;
+  border-top: 1px solid var(--borderColor);
+  text-decoration: none;
+  background-color: var(--bgColorQuestion);
+  color: inherit; /* blue colors for links too */
+}
+/* Make menu links pretend to not be links */
+a.menu-link {
+  color: inherit;
+  text-decoration: inherit;
+}
+.menu-link:hover {
+  background-color: var(--bgColorPage);
+}
+
+.menu-link:first-child {
+  border-top: none;
+}
+
+.menu-link:last-child {
+  border-bottom: none;
+}
```

Comment:
I really hope someone will come along and do proper styling of everything, so I don't have to...

Reply:
IMHO if there is no link to GH issue description and/or Figma, then there is no "proper" styling to implement 🤷

Reply:
From #285 hidden to the far right of linked [Figma file](https://www.figma.com/file/ipifwmO1KYfqakpl3srFIL/2023-08-08-AI-Safety-ccc-ccstan99?type=design&node-id=90-2047&mode=design)
<img width="673" alt="Screenshot 2023-11-02 at 1 19 18 AM" src="https://github.com/StampyAI/stampy-ui/assets/38997995/879b700e-d019-4682-a1b4-53ade74b10a0">
<img width="671" alt="Screenshot 2023-11-02 at 1 21 27 AM" src="https://github.com/StampyAI/stampy-ui/assets/38997995/833e300f-8fd7-4e4b-a909-a23fc66dd443">